spec/models/payment_intent_spec.rb

```diff
@@ -91,4 +91,16 @@
       it_behaves_like '#update PaymentIntent to incompatible status', 'legacy_failure', 'failed', 'succeeded'
     end
   end
+
+  describe 'update status and charges' do
+    let(:intent) { create(:payment_intent, :pending) }
+    let(:mock_update) { double("StripeUpdate", status: "requires_capture", last_payment_error: nil) }
```

Comment:
What is `"StripeUpdate"`?

Reply:
Okay, then perhaps could you add a comment that the name of the mocked class you chose (`"StripeUpdate"`) is arbitrary and (somewhat) meaningless for this test?

Reply:
Done